src/sentry/api/endpoints/organization_eventid.py

```diff
@@ -47,33 +47,22 @@ def get(self, request, organization, event_id):
                 'id', 'slug'))
 
         try:
-            event = Event.objects.filter(event_id=event_id,
-                                         project_id__in=project_slugs_by_id.keys())[0]
+            event = eventstore.get_events(filter_keys={
+                'project_id': project_slugs_by_id.keys(),
+                'event_id': event_id,
+            }, limit=1)[0]
         except IndexError:
```

Comment:
can you actually get IndexError from eventstore ?

Reply:
No, you'll always get a list, the index error is for when there's no matches and you try to do `[0]` on the empty array